package.json

```diff
@@ -7,6 +7,8 @@
   "engines": {
     "node": ">=6.13.0"
   },
+  "proxy": "http://localhost:1000/",
```

Comment:
Same question applies here.

Reply:
this one didn't work but I am leaving it here and going to ask during OH